src/services/WizardService.ts

```diff
@@ -2,10 +2,10 @@ import { autoinject } from "aurelia-framework";
 import { Router, RouterConfiguration, NavigationInstruction, RouterEvent } from "aurelia-router";
 import { EventAggregator } from "aurelia-event-aggregator";
 
-export interface IWizardState {
+export interface IWizardState<Data = any> {
   stages: Array<IWizardStage>;
   indexOfActive: number;
-  registrationData: IRegistrationData;
+  registrationData: Data;
```

Comment:
@paweljanicki 
Could you provide a sample Interface for `Data` other than `IDealRegistrationData`?

I remember you mentioned, that `Data` can be anything.
Currently in the code, though, only `IDealRegistrationData` is ued for the `Data` generic here.
So I'm wondering about the future use case of the `Data` generic.